melodic/distribution.yaml

```diff
@@ -3970,6 +3970,17 @@ repositories:
       url: https://github.com/JdeRobot/assets.git
       version: kinetic-devel
     status: developed
+  jderobot_camviz:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/JdeRobot/CamViz-release.git
+      version: 0.1.0-1
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/JdeRobot/CamViz.git
```

Comment:
The sources here have a LICENSE file that says "GPLv3", but the package.xml has "BSD" listed as the license.  One of those is wrong; please make them consistent.

Reply:
I have changed the BSD license to GPLv3 in package.xml. Please review the changes.